### PR TITLE
docs: update ecosystem

### PR DESCRIPTION
**Pull request by Ares-X**
add hexo theme support from freemind.bithack

### CODE REVIEW

**Comment by Mister-Hope** [comment]
1. Pass the linter
2. We have en and zh docs, and I saw your project is placing en readme with links to zh readme, so please add them both.

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
See comment above

**Comment by Ares-X** [comment]
done

File: docs/src/advanced/ecosystem.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Others:
 - 🐴 [Stun](https://github.com/liuyib/hexo-theme-stun)
 - 🤖 [Yun](https://github.com/YunYouJun/hexo-theme-yun)
 - 🦄 [Unicorn](https://github.com/Uzizkp/hexo-theme-unicorn)
-
+- 💀 [freemind.bithack](https://github.com/Ares-X/hexo-theme-freemind.bithack)
```
Comment (Mister-Hope) [inline docs/src/advanced/ecosystem.md:66]:
```suggestion
- 💀 [freemind.bithack](https://github.com/Ares-X/hexo-theme-freemind.bithack)

```

**Pull request merged**